sormas-api/src/main/java/de/symeda/sormas/api/event/EventExportDto.java

```diff
@@ -46,6 +94,7 @@
 	private String evolutionComment;
 	private String eventTitle;
 	private String eventDesc;
+	private EventGroupsIndexDto eventGroups;
```

Comment:
I think latest event group and event group count should be split in the export because that allows more flexibility with sorting